examples/helm/ceph/values.yaml

```diff
@@ -136,6 +143,13 @@ resources:
     limits:
       memory: "1Gi"
       cpu: "1000m"
+  exporter:
```

Comment:
I do not have any knowledge what would be the best values for exporters, I've put some values as the installation is failing without it.